docs/blog/v5-getting-started.md

```diff
@@ -0,0 +1,313 @@
+```
+
+## Use operator
+
+Easiest way to get started is to look in our [examples](../../examples/) which contains multiple examples on how to configure a grafana instance.
+
+Other then looking at the examples it's also good to use one of the most underrated command in Kubernetes, `explain`.
+For example, `kubectl explain grafanadashboard.spec` will give you insights on how you can configure the grafanadashboard.
+
+### Basic example
+
+Lets start with the basic example, this isn't something that you should use in production due to how we define the admin password but it's a simple way of getting started.
+
+```shell
+kubectl apply -f https://raw.githubusercontent.com/grafana-operator/grafana-operator-experimental/master/examples/basic/resources.yaml
+```
+
+Notice the label on the grafana resource, this is the one that GrafanaDashboard will use to find this instance.
+
+```.yaml
+apiVersion: grafana.integreatly.org/v1beta1
+kind: Grafana
+metadata:
+  name: grafana
+  labels:
+    dashboards: "grafana"
+spec:
+  config:
+    log:
+      mode: "console"
+    auth:
+      disable_login_form: "false"
+    security:
+      admin_user: root
+      admin_password: secret
+```
+
+And this is how the GrafanaDashboard, looks like.
+Sadly there is no good webhook solution or similar in grafana so we have to continuously poll the grafana API and see if there have been any changes made to the dashboard.
+This is the same way we did it in version 4.
+
+We also need to set the instanceSelector to find the grafana instance that this dashboard should be applied to.
+
+```.yaml
+apiVersion: grafana.integreatly.org/v1beta1
+kind: GrafanaDashboard
+metadata:
+  name: grafanadashboard-sample
+spec:
+  resyncPeriod: 30s
+  instanceSelector:
+    matchLabels:
+      dashboards: "grafana"
+  json: >
+    {
+      "id": null,
+      "title": "Simple Dashboard",
+      "tags": [],
+      "style": "dark",
+      "timezone": "browser",
+      "editable": true,
+      "hideControls": false,
+      "graphTooltip": 1,
+      "panels": [],
+      "time": {
+        "from": "now-6h",
+        "to": "now"
+      },
+      "timepicker": {
+        "time_options": [],
+        "refresh_intervals": []
+      },
+      "templating": {
+        "list": []
+      },
+      "annotations": {
+        "list": []
+      },
+      "refresh": "5s",
+      "schemaVersion": 17,
+      "version": 0,
+      "links": []
+    }
+```
+
+For simplicity lets port-forward to the grafana-deployment that we have created.
+
+```shell
+kubectl port-forward deployment/grafana-deployment 3000
+```
+
+You should now be able to go to localhost:3000 in your browser and login with `username: root` and `password: secret`.
+
+But wait a second didn't we setup a kind cluster with ingress support?
+Yes we did, so in the next session lets use it.
+
+### Ingress example
+
+TODO write a few rows how to fix DNS.
```

Comment:
@weisdd here it would be great to see some of your DNS magic.
Would you be up for a short commit with it unless it's to painful to setup.